website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -79,6 +79,13 @@ The default values of some [ember-power-select](https://ember-power-select.com/d
     <br><br>
     <strong>Note</strong>: It cannot be overridden.
   </C.Property>
+  <C.Property @name="searchFieldPosition" @type="string">
+    Determines whether the search field, when enabled, renders either within the before options content or in the “trigger” area.
+    <br><br>
+    Set to `"before-options"` to fix a `PowerSelect` accessibility issue. (Rendering the search input within the trigger area creates a nested interaction.)
```

Comment:
Is there any more a11y-related content related to this that we would want to add within the SuperSelect docs?  
(See: [Docs preview](https://hds-website-git-hds-4254-super-select-search-p-2680bc-hashicorp.vercel.app/components/form/super-select))

@MelSumner @shleewhite

Reply:
I don't think we need to necessarily link out to anything, you could just elaborate a bit more. Something like this:

```suggestion
    Set to `"before-options"` to fix a `PowerSelect` accessibility issue. Rendering the search input within the trigger is not complaint because it nests an interactive element within another interactive element.
```